tests/test_pyrflx.py

```diff
@@ -13,27 +17,34 @@
 )
 from rflx.pyrflx import (
     EnumValue,
-    Field,
     IntegerValue,
-    Message,
+    MessageValue,
     NotInitializedError,
     OpaqueValue,
     Package,
     PyRFLX,
     TypeValue,
 )
+from rflx.pyrflx.bitstring import Bitstring
+from rflx.pyrflx.typevalue import ArrayValue
 
 
-# pylint: disable=too-many-public-methods
 class TestPyRFLX(unittest.TestCase):
+    # pylint: disable=too-many-public-methods, too-many-instance-attributes, too-many-lines
+
     testdir: str
     specdir: str
+    package_tlv_checksum: Package
     package_tlv: Package
     package_ethernet: Package
     package_tls_record: Package
     package_tls_alert: Package
     package_icmp: Package
     package_test_odd_length: Package
+    package_ipv4: Package
+    package_array_typevalue: Package
+    package_array_nested_msg: Package
+    package_udp: Package
```

Comment:
@jklmnn what do you think about this? If i remove this i am getting a lot of errors and i am not really familiar with how the test class is set up.

Reply:
MyPy needs this as it would otherwise cause MyPy to not recognize the packages as attributes. For MyPy attributes can only be defined inside of `__init__`. It cannot deduce that `setUpClass` will be called before these attributes are used.